### ISSUE TITLE
Support animated gif avatars? 

### ISSUE THREAD

**Issue by shleeable** [pixelfed/pixelfed#3542]
Thanks.

Maybe create a OPTIN_AVATAR_GIF toggle?

**Comment by foxyseta** [comment]
Agreed. Even if Pixelfed does not allow its own users to have animated GIF avatars, other ActivityPub-compatible software does, and Pixelfed currently does not even render those as still images either.
